lib/ansible/modules/files/blockfromfile.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Evan Kaufman <evan@digitalflophouse.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: blockfromfile
+author:
+  - Evan Kaufman (@EvanK)
+version_added: "2.5"
+short_description: Search file from remote node using a provided regular expression
```

Comment:
So I think it's weird to state *"file from a remote node"* since that's basically what every module does. The exception is what needs to be mentioned. Maybe that's why `src` was not a good default parameter name after all.

I'd state here:
```yaml
short_description: Find patterns in files
```
And the module name is probably not what people would expect to get.